fabric-crash-report-info-v1/src/main/java/net/fabricmc/fabric/mixin/crash/report/info/MixinCrashReport.java

```diff
@@ -64,8 +66,7 @@ private static void appendMods(StringBuilder modString, int depth, TreeSet<ModCo
 			modString.append(mod.getMetadata().getVersion().getFriendlyString());
 
 			if (!mod.getContainedMods().isEmpty()) {
-				TreeSet<ModContainer> childMods = new TreeSet<>(Comparator.comparing(m -> m.getMetadata().getId()));
-				childMods.addAll(mod.getContainedMods());
+				ArrayList<ModContainer> childMods = new ArrayList<>(mod.getContainedMods());
```

Comment:
Should we sort the child mods by id, too?

Reply:
That's already done at the beginning of `appendMods` I believe (also for recursive calls).